hexa/workspaces/graphql/schema.graphql

```diff
@@ -118,11 +141,24 @@ input InviteWorkspaceMemberInput {
   role: WorkspaceMembershipRole!
 }
 
+input DeleteWorkspaceMemberInput {
+  workspaceId: String!
+  membershipId: String!
+}
+
+input UpdateWorkspaceMemberRoleInput {
+  workspaceId: String!
+  membershipId: String!
+  role: WorkspaceMembershipRole!
+}
+
 extend type Mutation {
-  createWorkspace(input: CreateWorkspaceInput!): CreateWorkspaceResult! @login_required
-  updateWorkspace(input: UpdateWorkspaceInput!): UpdateWorkspaceResult! @login_required
-  deleteWorkspace(input: DeleteWorkspaceInput!): DeleteWorkspaceResult! @login_required
-  inviteWorkspaceMember(input: InviteWorkspaceMemberInput!): InviteWorkspaceMemberResult! @login_required
+  createWorkspace(input: CreateWorkspaceInput!): CreateWorkspaceResult!
+  updateWorkspace(input: UpdateWorkspaceInput!): UpdateWorkspaceResult!
+  deleteWorkspace(input: DeleteWorkspaceInput!): DeleteWorkspaceResult!
+  inviteWorkspaceMember(input: InviteWorkspaceMemberInput!): InviteWorkspaceMemberResult!
+  updateWorkspaceMemberRole(input: UpdateWorkspaceMemberRoleInput!):UpdateWorkspaceMemberRoleResult!
```

Comment:
WDYT of just naming it `updateWorkspaceMember` ? Not sure if we will not add other fields aside the `role`

Reply:
Yes, It's better you right.